### PR TITLE
UKIS 1_0001 Increase test coverage of the downstream data

### PR DESCRIPTION
**Pull request by yoganandkunche**
### What is the context of this PR?

Dumps the answers that are going to be submitted and compares with the answers that were entered for UKIS

### How to review 

Run the integration test.

### CODE REVIEW

File: tests/integration/ukis/test_ukis_submission_data.py [hunk 1]
```diff
@@ -0,0 +1,448 @@
+
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestUkisSubmissionData(IntegrationTestCase):
+
+    def test_mwss_data_matches_expected_1_0001(self):
```
Comment (collisdigital) [inline tests/integration/ukis/test_ukis_submission_data.py]: test name is ukis

File: tests/integration/ukis/test_ukis_submission_data.py [hunk 2]
```diff
@@ -0,0 +1,448 @@
...
+        self.post(form_data)
+
+        self.post(post_data={'significant-events-established-answer': 'Yes',
+                             'significant-events-turnover-increase-answer': 'No',
+                             'significant-events-turnover-decrease-answer': 'Yes'})
+
+        # We have completed General Business Information section
+        self.assertInPage('>General Business Information</')
+        self.assertInPage('You have successfully completed this section')
+        self.post(action='save_continue')
```
Comment (collisdigital) [inline tests/integration/ukis/test_ukis_submission_data.py]: The default actions is `save_continue` so really you don't need to pass `action` here at all; but if you prefer to keep it so it is clearer that's fine.

File: tests/integration/ukis/test_ukis_submission_data.py [hunk 3]
```diff
@@ -0,0 +1,448 @@
...
+        # We proceed to the questionnaire
+        self.post(action='start_questionnaire')
+
+        # We are in the Questionnaire & We are in General Business Information section
+        self.assertInPage('>UK Innovation Survey</')
+        self.assertInPage('General Business Information')
+        self.assertInPage('>Save and continue<')
+
+        # In General Business Information section
+        form_data = {'geographic-markets-answer': ['UK regional within approximately 100 miles of this business',
```
Comment (collisdigital) [inline tests/integration/ukis/test_ukis_submission_data.py]: formatting could be better here, i.e. one item per line etc.

File: tests/integration/ukis/test_ukis_submission_data.py [hunk 4]
```diff
@@ -0,0 +1,448 @@
...
+        self.post(form_data)
+        self.post(post_data={'new-goods-services-innovations-answer': 'Yes'})
+        self.post(post_data={'goods-services-innovations-new-answer': 'Yes'})
+        self.post(post_data={'percentage-turnover-2016-market-new-answer': '25',
+                             'percentage-turnover-2016-business-new-answer': '25',
+                             'percentage-turnover-2016-improvement-answer': '25',
+                             'percentage-turnover-2016-modified-answer': '24'})
+
+        # We have completed Goods and Services Innovation section
+        self.assertInPage('>Goods and Services Innovation</')
```
Comment (collisdigital) [inline tests/integration/ukis/test_ukis_submission_data.py]: What do you think about putting each of these groups into a separate function?

File: tests/integration/ukis/test_ukis_submission_data.py (hunk 3 above)
Comment (yoganandkunche) [reply tests/integration/ukis/test_ukis_submission_data.py]: I have improved formatting.

File: tests/integration/ukis/test_ukis_submission_data.py (hunk 1 above)
Comment (yoganandkunche) [reply tests/integration/ukis/test_ukis_submission_data.py]: changed it back to ukis

File: tests/integration/ukis/test_ukis_submission_data.py (hunk 4 above)
Comment (yoganandkunche) [reply tests/integration/ukis/test_ukis_submission_data.py]: Done.

File: tests/integration/ukis/test_ukis_submission_data.py [hunk 5]
```diff
@@ -0,0 +1,525 @@
...
+                "type": "uk.gov.ons.edc.eq:surveyresponse",
+                "tx_id": tx_id
+            }
+        }
+        return expected_downstream_data
+
+    def complete_survey(self, eq_id, form_type_id):
+        self.launchSurvey(eq_id, form_type_id, roles=['dumper'])
+
+        self.landing_page()
```
Comment (collisdigital) [inline tests/integration/ukis/test_ukis_submission_data.py:176]: Much clearer :+1:

**Review by collisdigital — APPROVED** [review]

**Review by dcdarrell9 — APPROVED** [review]

**Comment by collisdigital** [comment]
Nice signed commits @yoganandkunche  👍

**Pull request merged**